src/french.js

```diff
@@ -1,5 +1,5 @@
 const allInfo = {
-  tense: {
+  Tense: {
```

Comment:
I think it is better not to. Each mood has its own set of tenses. If I separate this in a list of moods and a list of tenses separately, it could lead to wrong combinations of the two and lead to errors.